docs/user/custom-resources/apirule/v2alpha1/04-10-apirule-custom-resource.md

```diff
@@ -104,6 +104,44 @@ See an example of a valid **rules.path** order, listed from the most specific to
 - `/anything/{**}`
 - `/{**}`
 
+Understanding the relationship between paths and methods in a rule is crucial to avoid unexpected behavior. For instance, the following APIRule configuration excludes the `POST` and `GET` methods for the path `/anything/one` with `noAuth`. This happens because the rule with the path `/anything/{**}` shares at least one common method (`GET`) with a preceding rule.
```

Comment:
```suggestion
Understanding the relationship between paths and methods in a rule is crucial to avoid unexpected behavior. For example, the following APIRule configuration excludes the `POST` and `GET` methods for the path `/anything/one` with `noAuth`. This happens because the rule with the path `/anything/{**}` shares at least one common method (`GET`) with a preceding rule.
```